<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Medicamentos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Sistema-Hidrocolon/views/InterfaceImages/313969136_5709585199110797_1271018789977176534_n.png</normaloff>../../Sistema-Hidrocolon/views/InterfaceImages/313969136_5709585199110797_1271018789977176534_n.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>125</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>650</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
font: bold 12pt &quot;Nunito&quot;;</string>
            </property>
            <property name="text">
             <string>PDF Bitácora</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>55</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(111, 111, 111);
font: bold 7pt &quot;Nunito&quot;;</string>
            </property>
            <property name="text">
             <string>Ingrese el rango de fechas que desea obtener de la bitácora.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame_11">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_18">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
            </property>
            <property name="text">
             <string>Fecha Inicio:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="fecha_inicio">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2025</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_10">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_17">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
            </property>
            <property name="text">
             <string>Fecha Final:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="fecha_final">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2025</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2" rowspan="2">
        <widget class="QPushButton" name="pushButton_8">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 1px solid #ccc;
    border-radius: 10px;           /* Bordes redondeados con radio de 10 píxeles */
    padding: 5px;
	font: bold 9pt &quot;Nunito&quot;;                  /* Espaciado interno del botón */
}

QPushButton:hover{
font: bold 9pt &quot;Nunito&quot;;
background-color: #f0f0f0; /* Color de fondo deseado */
}
</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Sistema-Hidrocolon/views/InterfaceImages/eliminar.png</normaloff>../../Sistema-Hidrocolon/views/InterfaceImages/eliminar.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>189</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QPushButton" name="btn_agregar_medi">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 1px solid #ccc;
    border-radius: 10px;           /* Bordes redondeados con radio de 10 píxeles */
    padding: 5px;
	font: bold 9pt &quot;Nunito&quot;;                  /* Espaciado interno del botón */
}

QPushButton:hover{
font: bold 9pt &quot;Nunito&quot;;
background-color: #f0f0f0; /* Color de fondo deseado */
}
</string>
         </property>
         <property name="text">
          <string>Agregar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Sistema-Hidrocolon/views/InterfaceImages/lista-de-verificacion (1).png</normaloff>../../Sistema-Hidrocolon/views/InterfaceImages/lista-de-verificacion (1).png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
